changelog.json

```diff
@@ -1,5 +1,8 @@
 {
   "releases": {
+    "2.8.2-beta2": [
+      "[Fixed] Lists of commits, repositories or changes don't disappear after switching between apps"
```

Comment:
Should we add an improvement note for the Thank you work?
"Thank you note language improved and retrieves all past release notes to retroactively thank all contributors"

Reply:
Done @tidy-dev ! 😄